src/community/pom.xml

```diff
@@ -230,7 +230,7 @@
         <module>gwc-distributed</module>
         <module>jdbcstore</module>
         <module>gdal</module>
-        <module>gwc-s3</module>
+        <!-- <module>gwc-s3</module> -->
```

Comment:
Why this change?

Reply:
hmmm that is a mistake, needs to be undone